spec/mailers/previews/report_mailer_preview.rb

```diff
@@ -23,6 +23,21 @@ def monthly_key_metrics_report
     )
   end
 
+  def protocols_report
```

Comment:
this is covered by a spec that automatically iterates over all the preview methods right?

Reply:
yes it is, i checked! it's here https://github.com/18F/identity-idp/blob/main/spec/mailers/previews/report_mailer_preview_spec.rb#L9